vochain/offchaindatahandler/offchaindatahandler.go

```diff
@@ -42,6 +45,7 @@ var itemTypesToString = map[int]string{
 // offchain data (usually on IPFS).
 type OffChainDataHandler struct {
 	vochain       *vochain.BaseApplication
+	indexer       *indexer.Indexer
```

Comment:
i don't like this, but i can't think of a better option right now to communicate offchaindatahandler and indexer

Reply:
maybe a (cleaner?) alternative would be to add a new Event in EventListeners that is "MetadataReady" or something like this, that indexer can subscribe (and trigger the update in the db) and offchaindatahandler can trigger the event when the metadata is downloaded